src/custom/Feedback/FeedbackButton.tsx

```diff
@@ -0,0 +1,246 @@
+import { Checkbox } from '@mui/material';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Tooltip from '@mui/material/Tooltip';
+import Typography from '@mui/material/Typography';
+import React, { useRef, useState } from 'react';
+import {
+  CalenderIcon,
+  CloseIcon,
+  FeedbackIcon,
+  IdeaIcon,
+  QuestionIcon,
+  SuccessIcon
+} from '../../icons';
+import { ModalCard } from '../ModalCard';
+import {
+  CloseButton,
+  Container,
+  FeedbackButton,
+  FeedbackForm,
+  FeedbackMessage,
+  FeedbackMiniIcon,
+  FeedbackOptionButton,
+  FeedbackOptions,
+  FeedbackSubmitButton,
+  FeedbackTextArea,
+  MeetWrapper,
+  StyledLink,
+  StyledTextArea
+} from './style';
+
+const tooltipContent = (
+  <MeetWrapper>
+    Some account and system information may be sent to Layer5. We will use it to fix problems and
+    improve our services, subject to our{' '}
+    <StyledLink target="_blank" href="https://layer5.io/company/legal/privacy">
+      Privacy Policy
+    </StyledLink>{' '}
+    and{' '}
+    <StyledLink target="_blank" href="https://layer5.io/company/legal/terms-of-service">
+      Terms of Service
+    </StyledLink>
+    . We may email you for more information or updates.
+  </MeetWrapper>
+);
+
+interface FeedbackDataItem {
+  icon: JSX.Element;
+  label: string;
+  placeholder: string;
+  isTextInput: boolean;
+  innerComponent?: JSX.Element;
+}
+
+const feedbackData: FeedbackDataItem[] = [
+  {
+    icon: <FeedbackIcon />,
+    label: 'Issue',
+    placeholder: 'I’m having an issue with...',
+    isTextInput: true
+  },
+  {
+    icon: <IdeaIcon />,
+    label: 'Suggestion',
+    placeholder: 'I have a suggestion about...',
+    isTextInput: true
+  },
+  {
+    icon: <CalenderIcon />,
+    label: 'Meet Request',
+    placeholder: 'You have a suggestion3?',
```

Comment:
@captain-Akshay what is this? .... I'll tell you: it's not what I requested. Revisit the mockup.

Reply:
that placeholder is not used anywhere it's just a prop the whole component doesn't render it placeholder unless text field true and false is specified.

Reply:
Though removed it now